components/tidb_query/src/codec/datum.rs

```diff
@@ -124,7 +124,7 @@ impl Datum {
             Datum::Bytes(ref bs) => self.cmp_bytes(ctx, bs),
             Datum::Dur(d) => self.cmp_dur(ctx, d),
             Datum::Dec(ref d) => self.cmp_dec(ctx, d),
-            Datum::Time(ref t) => self.cmp_time(ctx, t),
+            Datum::Time(ref t) => self.cmp_time(ctx, *t),
```

Comment:
Seems the `ref` can be removed because of `Time` has been implemented trait `Copy`.

Reply:
Done